code/game/gamemodes/wizard/wizard.dm

```diff
@@ -135,30 +135,39 @@
 	qdel(wizard_mob.r_store)
 	qdel(wizard_mob.l_store)
 
+	if(!isplasmaman(wizard_mob))
```

Comment:
More readable if you turn this around.
If plasmaman do special case
else do normal case